src/tests/testRock.cc

```diff
@@ -97,14 +103,24 @@ TestRock::setUp()
     if (Ipc::Mem::Segment::BasePath == nullptr)
         Ipc::Mem::Segment::BasePath = ".";
 
+    // diff reduction; TODO: inline
+    finishStartup();
+}
+
+void
+TestRock::setUp()
+{
+    CPPUNIT_NS::TestFixture::setUp();
+
+    if (0 > system ("rm -rf " TESTDIR))
+        throw std::runtime_error("Failed to clean test work directory");
```

Comment:
This is old code, in the old per-test-case initialization sequences. No changes here despite what the diff misrepresents.